#!/usr/bin/env python
PACKAGE = "euclid_arduino_wrapper"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("backward_mult",    double_t,    0, "Backward Multiplier", 1,  0, 100)
gen.add("forward_mult",    double_t,    0, "Forward Multiplier", 1,  0, 100)
gen.add("rotation_mult",    double_t,    0, "Rotaion Multiplier", 1,  0, 100)
gen.add("left_gain",    double_t,    0, "Left Gain", 1,  0, 100)
gen.add("right_gain",    double_t,    0, "Right Gain", 1,  0, 100)

exit(gen.generate(PACKAGE, "euclid_arduino_wrapper", "EuclidMotorConfiguration"))
